styles.css

```diff
@@ -0,0 +1,105 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 2vh 5vw;
+}
+
+.vector_container {
+  display: flex;
+  flex-direction: column;
+}
+
+.welcome {
+  font-family: Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 20px;
+  line-height: 28px;
+}
+
+.vector {
+  fill: rgb(0, 0, 0);
+  stroke-width: 1;
+  stroke: rgb(0, 0, 0);
+  width: 40px;
+  height: 3px;
+}
+
+.vectSvg {
+  width: 40px;
+  height: 3px;
+}
+
+.vector:hover {
+  fill: rgb(55, 12, 224);
+  stroke-width: 1;
+  stroke: rgb(55, 12, 224);
+}
+
+body {
+  display: flex;
+  flex-direction: column;
+  align-items: space-between;
+  height: 100vh;
+  margin: 0 0 0 0;
+  padding: 0;
+}
+
+.pageSection {
+  width: 60vw;
+  flex: 1;
+}
+
+.pageSection h1 {
+  text-align: center;
+}
+
+.stText {
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  font-family: 'Inter', sans-serif;
+}
+
+.ndText {
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: green;
+  font-family: 'Inter', sans-serif;
+}
+
+.text {
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+}
+
+.mainSection {
+  flex: 1;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-image: url("header-illsutration-mobile.png");
+  background-repeat: no-repeat;
+  background-size: 100vw 100vh;
+  margin: 0;
+  padding: 0;
+}
+
+.socialMedia {
+  display: flex;
+  justify-content: space-between;
+  list-style-type: none;
+}
```

Comment:
- [ ] Kindly as requested by the previous review, the size and the margin to your icons are not match with the Figma template. remember You must stick to the design as much as possible (e.g. font, colors, images, tests, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).